src/clientManager.js

```diff
@@ -25,14 +25,8 @@ ClientManager.prototype.init = function(identHandler, sockets) {
 	}
 };
 
-ClientManager.prototype.findClient = function(name, token) {
-	for (var i in this.clients) {
-		var client = this.clients[i];
-		if (client.name === name || (token && token === client.config.token)) {
-			return client;
-		}
-	}
-	return false;
+ClientManager.prototype.findClient = function(name) {
+	return this.clients.find(u => u.name === name);
```

Comment:
Client now sends name+token combo instead of just token, so we can find the user and then compare the token. Previous version iterated all users and compared tokens.